tools/swift-format/Sources/Rules/NeverUseImplicitlyUnwrappedOptionals.swift

```diff
@@ -6,13 +6,51 @@ import SwiftSyntax
 ///
 /// Certain properties (e.g. `@IBOutlet`) tied to the UI lifecycle are ignored.
 ///
-/// This rule does not apply to test code, defined as code which matches one or more of:
-///   * Parent directory named "Tests"
+/// This rule does not apply to test code, defined as code which:
 ///   * Contains the line `import XCTest`
 ///
+/// TODO: Create exceptions for other UI elements (ex: viewDidLoad)
+///
 /// Lint: Declaring a property with an implicitly unwrapped type yields a lint error.
 ///
 /// - SeeAlso: https://google.github.io/swift#implicitly-unwrapped-optionals
 public final class NeverUseImplicitlyUnwrappedOptionals: SyntaxLintRule {
+  
+  // Checks if "XCTest" is an import statement
+  public override func visit(_ node: ImportDeclSyntax) {
+    var iterator = node.path.makeIterator()
+    while let component = iterator.next() {
+      if component.name.text == "XCTest" {
+        context.importsXCTest = true
+      }
+    }
+  }
+
+  public override func visit(_ node: VariableDeclSyntax) {
+    if !context.importsXCTest {
```

Comment:
Nitpick: `guard context.importsXCTest else { return }`

This avoids nesting the function body.